src/lib.rs

```diff
@@ -680,6 +667,11 @@ impl<V: fmt::Debug> fmt::Debug for VecMap<V> {
 
         write!(f, "}}")
     }
+
+    #[cfg(feature = "nightly")]
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_map().entries(self.iter()).finish()
+    }
```

Comment:
It's not faster, but it'll give us pretty-printing for free. Once 1.2.0 is released, we can remove the manual formatting.